File: apollo/src/main/java/io/muun/apollo/domain/errors/InvalidCharacterRecoveryCodeError.java
package io.muun.apollo.domain.errors;

import io.muun.apollo.external.UserFacingErrorMessages;

public class InvalidCharacterRecoveryCodeError extends UserFacingError {

    public InvalidCharacterRecoveryCodeError() {
        super(UserFacingErrorMessages.INSTANCE.invalidCharacterRecoveryCode());
    }
}

File: android/apolloui/src/main/java/io/muun/apollo/domain/errors/rc/InvalidCharacterRecoveryCodeError.kt
package io.muun.apollo.domain.errors.rc

import io.muun.apollo.data.external.UserFacingErrorMessages
import io.muun.apollo.domain.errors.ErrorClassification
import io.muun.apollo.domain.errors.UserFacingError

class InvalidCharacterRecoveryCodeError :
    UserFacingError(UserFacingErrorMessages.INSTANCE.invalidCharacterRecoveryCode()) {
    override val classification = ErrorClassification.EXPECTED
}
